### PR TITLE
Updated configs

### PR DESCRIPTION
**Pull request by iulianpascalau**
- changed ERD naming in configs to eGLD
- reduced the minimum stake value

### CODE REVIEW

File: cmd/node/config/systemSmartContractsConfig.toml [hunk 1]
```diff
@@ -11,15 +11,15 @@
     MaxNumberOfNodesForStake = 2169
     NodesToSelectInAuction = 2169
     UnJailValue = "2500000000000000000" #0.1% of genesis node price
-    ActivateBLSPubKeyMessageVerification = false #TODO set to true before mainnet
+    ActivateBLSPubKeyMessageVerification = false
 
 [ESDTSystemSCConfig]
-    BaseIssuingCost = "5000000000000000000000" #5000ERD
+    BaseIssuingCost = "5000000000000000000000" #5000 eGLD
```
Comment (iulianpascalau) [inline cmd/node/config/systemSmartContractsConfig.toml:17]: is this supposed to be 5000?

File: cmd/node/config/systemSmartContractsConfig.toml [hunk 2]
```diff
@@ -11,15 +11,15 @@
...
     OwnerAddress = "erd1932eft30w753xyvme8d49qejgkjc09n5e49w4mwdjtm0neld797su0dlxp"
     Disabled = false
 
 [GovernanceSystemSCConfig]
-	ProposalCost = "5000000000000000000000" #5000ERD
+	ProposalCost = "5000000000000000000000" #5000 eGLD
```
Comment (iulianpascalau) [inline cmd/node/config/systemSmartContractsConfig.toml:22]: is this supposed to be 5000?

File: cmd/node/config/systemSmartContractsConfig.toml (hunk 1 above)
Comment (sasurobert) [reply cmd/node/config/systemSmartContractsConfig.toml:17]: 5 should be enough

File: cmd/node/config/systemSmartContractsConfig.toml (hunk 2 above)
Comment (sasurobert) [reply cmd/node/config/systemSmartContractsConfig.toml:22]: 5 should be enough

**Review by sasurobert — APPROVED** [review]
base issuing cost can be changed when enabled

File: cmd/node/config/systemSmartContractsConfig.toml (hunk 1 above)
Comment (iulianpascalau) [reply cmd/node/config/systemSmartContractsConfig.toml:17]: will keep as it is, for now

File: cmd/node/config/systemSmartContractsConfig.toml (hunk 2 above)
Comment (iulianpascalau) [reply cmd/node/config/systemSmartContractsConfig.toml:22]: will keep as it is, for now

**Review by AdoAdoAdo — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]

**Pull request merged**